src/views/Project/index.jsx

```diff
@@ -155,6 +156,23 @@ export default class Project extends Component {
       this.load();
     }
   }
+
+  handleFetchBugFirstComment = async id => {
+    try {
+      const {
+        data: { comments },
+      } = await this.props.client.query({
```

Comment:
I think you can `import { withApollo } from 'react-apollo';` and use @withApollo on top of the class to have access to `this.props.client`. You can then remove `<ApolloConsumer>` from `src/App.jsx`.